docs/guides/jupyter.rst

```diff
@@ -0,0 +1,398 @@
+         }
+
+Notice that the Markdown format does not store the outputs of the computation.
+nbsphinx will automatically execute notebooks without outputs,
+so in your HTML documentation they appear as complete.
+
+.. _notebook-galleries:
+
+Creating galleries of examples using notebooks
+----------------------------------------------
+
+`nbsphinx`_ has support for :doc:`creating thumbnail galleries from a list of Jupyter
+notebooks <nbsphinx:subdir/gallery>`.
+This functionality relies on `Sphinx-Gallery <https://sphinx-gallery.github.io/>`_
+and extends it to work with Jupyter notebooks rather than Python scripts.
+
+To use it, you will need to install both nbsphinx and Sphinx-Gallery,
+and modify your ``conf.py`` as follows:
+
+.. code-block:: python
+   :caption: conf.py
+
+   extensions = [
+      'nbsphinx',
+      'sphinx_gallery.load_style',
+   ]
+
+After doing that, there are two ways to create the gallery:
+
+- From a reStructuredText source file, using the ``.. nbgallery::`` directive,
+  :ref:`as showcased in the
+  documentation <nbsphinx:/a-normal-rst-file.rst#thumbnail-galleries>`.
+- From a Jupyter notebook, adding a ``"nbsphinx-gallery"`` tag to the metadata of a cell.
+  Each editor has a different way of modifying the cell metadata (see figure below).
+
+.. figure:: /_static/images/guides/jupyterlab-metadata.png
+   :width: 80%
+   :align: center
+   :alt: Panel to modify cell metadata in JupyterLab
+
+   Panel to modify cell metadata in JupyterLab
+
+For example, this reST markup would create a thumbnail gallery
+with generic images as thumbnails,
+thanks to the Sphinx-Gallery default style:
+
+.. code-block:: rest
+
+   Thumbnails gallery
+   ==================
+
+   .. nbgallery::
+      notebooks/Example 1
+      notebooks/Example 2
+
+.. figure:: /_static/images/guides/thumbnail-gallery.png
+   :width: 80%
+   :align: center
+   :alt: Simple thumbnail gallery created using nbsphinx
+
+   Simple thumbnail gallery created using nbsphinx
+
+To see some examples of notebook galleries in the wild:
+
+- `poliastro <https://docs.poliastro.space/>`_ offers tools for interactive Astrodynamics in Python,
+  and features `several examples and how-to guides using
+  notebooks <https://docs.poliastro.space/en/stable/gallery.html>`_
+  and displays them in an appealing thumbnail gallery.
+  In addition, `poliastro uses unpaired MyST
+  Notebooks <https://github.com/poliastro/poliastro/tree/0.15.x/docs/source/examples>`_
+  to reduce repository size and improve integration with git.
+
+Background
+----------
+
+.. _nb-background:
+
+Existing relevant extensions
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+There are two main extensions that add support Jupyter notebooks as source files in Sphinx:
+nbsphinx_ and MyST-NB_. They have similar intent and basic functionality:
+both can read notebooks in ``.ipynb`` and additional formats supported by `jupytext`_,
+and are configured in a similar way.
```

Comment:
I think we already talked about them at the beginning of the document, does it make sense to rephrase this saying something like: "as we have seen, there two main extensions... let's jump into their differences" or similar?

I'm not sure if we presented the extensions previously but only mentioned that there are two ways of doing the guide. It may make sense to make this presentation earlier in the doc and here just refer to that?

Reply:
Yes, I will rephrase the beginning and this part so it makes more sense. Otherwise, this introduction comes "too late"